specs-go/config.go

```diff
@@ -51,17 +53,21 @@ type Process struct {
 	ApparmorProfile string `json:"apparmorProfile,omitempty" platform:"linux"`
 	// SelinuxLabel specifies the selinux context that the container process is run as. (this field is platform dependent)
 	SelinuxLabel string `json:"selinuxLabel,omitempty" platform:"linux"`
+
+	// InitialConsoleSize specifies the initial [h,w] of the console. (this field is platform dependent)
+	InitialConsoleSize [2]int `json:"initialConsoleSize,omitempty" platform:"windows"`
 }
 
-// User specifies Linux/Solaris specific user and group information for the container's
-// main process.
+// User specifies the user information for the container
 type User struct {
 	// UID is the user id. (this field is platform dependent)
 	UID uint32 `json:"uid" platform:"linux,solaris"`
 	// GID is the group id. (this field is platform dependent)
 	GID uint32 `json:"gid" platform:"linux,solaris"`
 	// AdditionalGids are additional group ids set for the container's process. (this field is platform dependent)
 	AdditionalGids []uint32 `json:"additionalGids,omitempty" platform:"linux,solaris"`
+	//  User is the user name. (this field is platform dependent)
+	User string `json:"user" platform:"windows"`
```

Comment:
Any specific reason why it's not username, and can we expect a groupname in the future ?